test/Github/Tests/ResultPagerTest.php

```diff
@@ -97,9 +101,12 @@ public function testFetch()
         $result = 'foo';
         $method = 'bar';
         $parameters = array('baz');
-        $api = $this->getMock('Github\Api\ApiInterface');
+        $api = $this->createMock('Github\Api\ApiInterface');
```

Comment:
You can use `getMockBuilder()` instead which works on all supported PHPUnit versions without triggering deprecations.